1/index.html

```diff
@@ -0,0 +1,485 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Task1</title>
+
+    <link rel="stylesheet" href="styles/main.css">
+
+
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+
+    <script src="index.js" defer></script>
+
+
+</head>
+<body>
+    <header class="container-fluid "></header>
+<main>
+    <div class="header-menu container-fluid d-flex">
```

Comment:
This div should be a <header> and the previous one (if you really need it) can be a div

Reply:
relocate it